scapy/contrib/igmpv3.py

```diff
@@ -67,12 +67,13 @@ class IGMPv3(IGMP):
                     ByteField("mrcode", 20),
                     XShortField("chksum", None)]
 
-    def float_encode(self, value):
+    def encode_maxrespcode(self):
         """Convert the integer value to its IGMPv3 encoded time value if needed.
```

Comment:
You need to update the docstring so that it reflects what this method actually does.

Reply:
Updated, thanks !